app/views/teacher_training_adviser/steps/completed.html.erb

```diff
@@ -15,6 +15,17 @@
         <p class="govuk-body">
           You must respond to the email within 6 weeks of receiving it otherwise you will have to sign up again.  
         </p>
+
+        <p class="govuk-body">
+        While you wait for your adviser, you can: 
+      <ul>
+        <li>Ask any immediate questions using the Get Into Teaching <a href="https://beta-getintoteaching.education.gov.uk/#talk-to-us">online chat</a></li>
```

Comment:
@MylesJarvis we need to avoid putting hard links to the beta domain in the TTA otherwise they'll break when we transition across and deprecate the service.

This will need a new Rails helper, I'll pick it up once I've finished something I'm in the middle of and update your PR.